### PR TITLE
assignment1 and 2

### PR DESCRIPTION
**Pull request by harshalkotkar**

### CODE REVIEW

File: HARSHAL/Reactjs/react_app/src/components/assignment2.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+export default class IncrementDecrement extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            count: 0,
+        };
+    }
+    changecountincrement = () => {
```
Comment (MKadeer17) [inline HARSHAL/Reactjs/react_app/src/components/assignment2.jsx:11]: This Ok method name, but try make method name like this:
easy to Understand.
```suggestion
    changeCountIncrement = () => {
```

**Review by MKadeer17 — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

File: HARSHAL/Reactjs/react_app/src/components/assignment1.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+
+class Currencyconvertor extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            amount: "",
```
Comment (Snehal-113) [inline HARSHAL/Reactjs/react_app/src/components/assignment1.jsx]: amount : 0

File: HARSHAL/Reactjs/react_app/src/components/assignment2.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        this.setState({
+            count: this.state.count + 1,
+        });
+        if(this.state.count>9) {alert("value greater than 10")}
+    };
+    changecountdecrement = () => {
+        this.setState({
```
Comment (Snehal-113) [inline HARSHAL/Reactjs/react_app/src/components/assignment2.jsx:18]: counter value must not be decremented after 0
and not be incremented after 10
put proper conditons

File: HARSHAL/Reactjs/react_app/src/components/assignment2.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+export default class IncrementDecrement extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            count: 0,
+        };
+    }
+    changecountincrement = () => {
+        this.setState({
```
Comment (Snehal-113) [inline HARSHAL/Reactjs/react_app/src/components/assignment2.jsx:12]: same as mentioned below

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Pull request merged**